config/src/main/java/com/yahoo/config/subscription/impl/JRTConfigRequester.java

```diff
@@ -217,7 +219,7 @@ private long calculateErrorTimeout() {
     private void handleFatallyFailed(JRTClientConfigRequest jrtReq,
                                      JRTConfigSubscription<ConfigInstance> sub, long delay) {
         if (sub.getState() != ConfigSubscription.State.OPEN) return;
-        fatalFailures++;
+        fatalFailures = true;
```

Comment:
... and transientFailures to false here?

Reply:
Yeah, I think that makes sense, I'll do it in the next PR.